python/tank/deploy/tank_command.py

```diff
@@ -47,9 +47,9 @@
 ###############################################################################################
 # Built in actions (all in the tank_commands sub module)
 
-BUILT_IN_ACTIONS = [setup_project.SetupProjectAction, 
+BUILT_IN_ACTIONS = [setup_project.SetupProjectAction,
                     setup_project_wizard.SetupProjectFactoryAction,
-                    core_upgrade.CoreUpgradeAction, 
+                    core_upgrade.CoreUpdateAction,
```

Comment:
I was also very careful not to update the status messages for that very reason, since they are public.